### PR TITLE
borgbackup: update to 1.1.4

### PR DESCRIPTION
**Pull request by yuroller**
_Motivation:_ Update package borgbackup to 1.1.4.
_Linked issues:_ https://github.com/SynoCommunity/spksrc/pull/2998 https://github.com/SynoCommunity/spksrc/issues/2351

### Checklist
- [] Build rule `all-supported` completed successfully
- [] Package upgrade completed successfully
- [x] New installation of package completed successfully

Tested on Synology DS212+

### CODE REVIEW

**Comment by Safihre** [comment]
Let's see if we can get it to pickup the .so files without including them in Python3.

**Comment by yuroller** [comment]
Thanks. borgbackup requires libacl which in turn requires libattr.
The package I provided has this .so libs problem, but it works on my synology unit: over the weekend I did a remote backup of some GB of data in order to test it.

File: spk/python3/Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ DEPENDS += cross/setuptools cross/pip cross/wheel
 DEPENDS += cross/cffi cross/bcrypt
 # Cross-compiled wheels
 DEPENDS += cross/lxml cross/pycrypto cross/pycurl cross/pyyaml
-DEPENDS += cross/msgpack-python cross/attr cross/acl cross/lz4
```
Comment (ymartin59) [inline spk/python3/Makefile]: I do not understand why removing `lz4` from Python 3... I may impact other packages running over Python 3.
What is the reason / motivation ?

**Comment by yuroller** [comment]
Lz4 was added because the previous version of borgbackup needed it and, as far as I know, no other package is using it.
Starting from borgbackup 1.1.4, libz4 is bundled with borgbackup, so it is not needed anymore in python3.

**Comment by ymartin59** [comment]
So it means it should run with already published Python 3 package. I will test and publish if OK.

**Comment by yuroller** [comment]
Yes, it should. But just if you lower the python3 required version in borgbackup package or else it would not install.

**Comment by ymartin59** [comment]
So please remove changes to `cross/python3` and change dependencies. When done, I will merge and publish.
Just a question as a confirmation: it is not expected to run as service, it is only a command line utility, right ?

**Comment by yuroller** [comment]
It is not a service: it runs from commandline.
About python3 dependencies: borgbackup requires libacl (which in turn requires libattr), liblz4 is not required anymore (it is bundled in borgbackup source package): I was not able to put those dependencies inside borg package, but instead had to put inside python3 package or else borg it is not able to find them.
Can somebody explain me why is it like that?

**Comment by ymartin59** [comment]
So please:
- remove changes to `cross/python3`
- remove useless `spk/borgbackup/src/dsm-control.sh` and `spk/borgbackup/src/installer.sh`
- remove package dependency to `SPK_DEPENDS = "python3>=3.5.2-6"`
When done, I will merge and publish.

**Comment by ymartin59** [comment]
@yuroller May you please review my request for changes from https://github.com/SynoCommunity/spksrc/pull/3196#issuecomment-371613824 - I really find it good so it will be merged and published

**Comment by ymartin59** [comment]
Superseed by #3280

**Pull request closed without merging**